rllib/env/single_agent_env_runner.py

```diff
@@ -176,13 +182,11 @@ def _sample_timesteps(
             for i in range(self.num_envs):
                 # TODO (sven): Maybe move this into connector pipeline
                 # (even if automated).
-                self._episodes[i].add_initial_observation(
-                    initial_observation=obs[i],
-                    initial_info=infos[i],
-                    # TODO (simon): Check, if this works for the default
-                    # stateful encoders.
-                    initial_state={k: s[i] for k, s in states.items()},
+                self._episodes[i].add_env_reset(
```

Comment:
Cleaner naming of these Episode methods:
* `add_env_reset`
* `add_env_step`

Both add to an episode the return values of those gym.Env calls.